_clanky/neziskovky-automatizace-dashboardu.md

```diff
@@ -0,0 +1,97 @@
+
+<!--more-->
+
+Už delší dobu existuje řada nástrojů, které i běžnému uživateli bez znalosti IT umožňují vytvořit si nad daty reportingovou nástěnku, tzv. dashboard. Tyto nástroje zvládají už i poloautomatickou aktualizaci, kdy častokrát stačí data jenom získat/stáhnout, zkopírovat, nahrát a dashboard se už sám aktualizuje. I takto zjednodušený proces se však stává příliš náročným, pokud potřebujeme data aktualizovat na denní nebo týdenní bázi.  Navíc, mnohokrát se stává, že i měsíční nebo roční aktualizace přináší pocit rutiny a zbytečně stráveného času.
+
+Pokud byste si chtěli ulehčit práci a vytvořit nad daty z [Národního katalogu otevřených dat][NKOD](NKOD) nebo jakéhokoliv jiného datového zdroje dashboard, který bude aktualizován i bez nutnosti trávit čas znovu nahráváním, kopírováním a úpravou souborů, máme pro Vás dobrou zprávu. Existuje způsob, jak se můžete k takovému automaticky aktualizovanému dashboardu dopracovat. Nevyžaduje navíc žádné znalosti programování, je zcela zdarma a není ani nijak zvlášť komplikovaný. Postup má sice velké množství kroků, avšak tyto jsou většinou velmi jednoduché a mnohé z nich je nutné udělat jenom jednou. Konečný proces, zbavený jednorázových kroků, je velice jednoduchý, efektivní a lehce replikovatelný na další a další datové sady. Následující text vám nabízíme jako menší kuchařku toho, jak se k takovému dashboardu dopracovat.
+
+Co budeme potřebovat:
+
+* textový editor kódu - nástroj pro psaní a a spouštění skriptů. Pro to aby jsme nemuseli data v dashboardu neustále ručně aktualizovat, budeme potřebovat skript, který za nás tuto práci bude sám pravidelně dělat. Textové editory jsou pak prostředí kde se takovéto skripty píšou a spouští.
+
+* programovací jazyk Python instalovaný na pc - skript který použijeme je psaný v jazyku Python a proto aby ho náš textový editor věděl interpretovat, je nutné mít tento jazyk nainstalovaný na PC. Můžeme si také pomoci analogií typů písma. Word, který jistě znáte, umí psát text v různých druzích písma jak například Arial nebo Calibri. Některé písma, jako například Quicksand, však nemá předinstalované a proto aby jsme mohli psát článek i v tomto druhu písma, je nutné si Quicksand nejdříve stáhnou a pak nainstalovat na PC. Word už pak sám rozpozná a nabídne nám možnost v tomto písmu psát. Stejně pak funguje i textový editor. S nainstalovaným jazykem Python získá možnost skripty v Pythonu interpretovat.
+
+* účet Google - účet Google jistě znáte a mnozí z vás ho už i máte, protože existuje široká paleta nástrojů které je možné s tímto účtem využívat jako například YouTube nebo Gmail. Pro naše účely je tento účet důležitý proto že nám umožní zdarma využívat také služby Google Data Studio pro vytvoření dashboardu a developerskou konzoli pro komunikaci skriptu s Google sheets.
+
+##Editor kódu
```

Comment:
Mezi `##` a názvem musí být mezera. Jinak to není nadpis:
![image](https://user-images.githubusercontent.com/718827/106703954-8d766a80-65eb-11eb-86d5-46c1697eeae6.png)